fe/fe-core/src/test/java/com/starrocks/http/rest/v2/TableSchemaActionTest.java

```diff
@@ -264,13 +476,12 @@ private static OlapTable newOlapTable(Long tableId, String tableName) {
 
     private static RestBaseResultV2<TableSchemaView> parseResponseBody(String body) {
         try {
-            System.out.println("resp: " + body);
-            Gson gson = new Gson();
-            return gson.fromJson(body, new TypeToken<RestBaseResultV2<TableSchemaView>>() {
+            System.out.println("Resp: " + body);
```

Comment:
not your fault, but please avoid printing log to stdout. use LOGGER instead.

Reply:
OK, it has been removed.